src/mumble/AudioInput.cpp

```diff
@@ -1024,10 +1026,16 @@ void AudioInput::encodeAudioFrame(AudioChunk chunk) {
 
 		if (ao) {
 			if (Global::get().s.bTxAudioCue) {
+				// Cancel active cue, if there is any
+				if (m_activeAudioCue && bIsSpeech != bPreviousVoice) {
```

Comment:
Maybe it would be best, if the conditions for the if-statements in this block were factored out as `const bool` variables before the actual ifs. That way, it would be clearer how the killing of the active cue relates to the a new cue being created.
Something like this:
```cpp
const bool playAudioOnCue = bIsSpeech && !bPreviousVoice;
const bool playAudioOffCue = !bIsSpeech && bPreviousVoice;
const bool stopActiveCue      = m_activeAudioCue && (playAudioOnCue || playAudioOffCue);
```
Technically, this might add the evaluation of an additional boolean expression, but this is a runtime penalty that I would be willing to introduce (if the compiler isn't smart enough to just optimize this out of the way anyway).